src/Microsoft.AspNetCore.Server.IISIntegration/IISMiddleware.cs

```diff
@@ -49,6 +47,19 @@ public IISMiddleware(RequestDelegate next, ILoggerFactory loggerFactory, IOption
 
             _next = next;
             _options = options.Value;
+
+            if (_options.ForwardWindowsAuthentication)
+            {
+                var auth = authentication.FirstOrDefault();
+                if (auth == null)
+                {
+                    throw new InvalidOperationException("IServiceCollection.AddAuthentication() is required to use Authentication.");
```

Comment:
Remove this exception for now, ForwardWindowsAuthentication is enabled by default.